Sources/StreamChatUI/ChatMessageList/MessageComposer/ChatMessageComposerVC.swift

```diff
@@ -97,6 +97,12 @@ open class _ChatMessageComposerVC<ExtraData: ExtraDataTypes>: _ViewController,
         setupInputView()
     }
 
+    override open func setUpLayout() {
+        super.setUpLayout()
+
+        view.embed(composerView)
```

Comment:
I don' think this established this as a rule but I personally don't like the `embed` helper and prefer using:
```
addSubview(view)
view.pin(to: self.layoutMarginsGuide)
```
since it's more explicit.

Reply:
Ah yes that is way better